src/pages/reagir-a-une-activite/1.tsx

```diff
@@ -20,7 +20,7 @@ const ReactionStep1 = () => {
   const { activity, createNewActivity, updateActivity } = React.useContext(ActivityContext);
   const selectRef = React.useRef<HTMLDivElement>(null);
   const [selectedActivity, setSelectedActivity] = React.useState<SelectedActivityInfos>({ id: null, type: null });
-  const activitiesTypes = [1, 2, 3, 9];
+  const activitiesTypes = [1, 2, 3, 5, 9];
```

Comment:
This works, but best would be to use the activity type constants instead of 'magic' numbers :)

Reply:
I'm on it :)

Reply:
Magic numbers are now connected to there constants